bin/publisher/tools.py

```diff
@@ -48,6 +69,33 @@ def get_rcc() -> str:
     return rcc_path
 
 
+def download_agent_cli() -> str:
+    working_dir = get_working_dir()
+    system = platform.system().lower()
+
+    if system == "windows":
+        url = f"https://cdn.sema4.ai/agent-cli/releases/{agent_cli_version}/windows64/agent-cli.exe"
+        agent_cli_exe = os.path.join(working_dir, "agent-cli.exe")
+    elif system == "darwin":
+        url = f"https://cdn.sema4.ai/agent-cli/releases/{agent_cli_version}/macos64/agent-cli"
+        agent_cli_exe = os.path.join(working_dir, "agent-cli")
+    elif system == "linux":
+        url = f"https://cdn.sema4.ai/agent-cli/releases/{agent_cli_version}/linux64/agent-cli"
+        agent_cli_exe = os.path.join(working_dir, "agent-cli")
+    else:
+        raise Exception("Unsupported OS")
```

Comment:
Not mandatory.
Might be better to have a have the URL in parts and reassemble it.
I'm just thinking that the agent-cli might come from `/beta/` and not `releases`